apps/apps.h

```diff
@@ -281,7 +282,11 @@ int set_cert_times(X509 *x, const char *startdate, const char *enddate,
             "Block size to pad TLS 1.3 records to."}, \
         {"debug_broken_protocol", OPT_S_DEBUGBROKE, '-', \
             "Perform all sorts of protocol violations for testing purposes"}, \
-        {"no_middlebox", OPT_S_NO_MIDDLEBOX, '-', "Disable TLSv1.3 middlebox compat mode" }
+        {"no_middlebox", OPT_S_NO_MIDDLEBOX, '-', \
+            "Disable TLSv1.3 middlebox compat mode" }, \
+        {"num_tickets", OPT_S_NUM_TICKETS, 'n', \
```

Comment:
This is a server-only option, would this change also define it in s_client?

Reply:
It should probably be changed to be a server-only option.

Reply:
s_client was reporting it as an error. But you're right it would be better if it was only defined in s_server - so I did that.